source/GameInterface/Services/CraftingService/Patches/CraftingDestructionPatches.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Common.Logging;
+using Common.Messaging;
+using GameInterface.Policies;
+using GameInterface.Services.CraftingService.Messages;
+using GameInterface.Services.ObjectManager;
+using HarmonyLib;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using TaleWorlds.CampaignSystem.GameState;
+using TaleWorlds.CampaignSystem.MapEvents;
+using TaleWorlds.CampaignSystem.Settlements;
+using TaleWorlds.CampaignSystem.Settlements.Buildings;
+using TaleWorlds.Core;
+using TaleWorlds.Localization;
+
+namespace GameInterface.Services.CraftingService.Patches
+{
+    internal class CraftingDestructionPatches
```

Comment:
This is technically a lifetime patch, could you move these into the Lifetime file?